operator/mysqlcontroller/connector.go

```diff
@@ -0,0 +1,34 @@
+package mysqlcontroller
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	exoscalesdk "github.com/exoscale/egoscale/v2"
+	exoscalev1 "github.com/vshn/provider-exoscale/apis/exoscale/v1"
+	"github.com/vshn/provider-exoscale/operator/pipelineutil"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type MySQLConnector struct {
+	Kube     client.Client
+	Recorder event.Recorder
```

Comment:
```suggestion
type connector struct {
	kube     client.Client
	recorder event.Recorder
```

nit - but that makes the visibility as small as necessary. Not sure what the general consensus in this repo is TBH :)